src/chigame/api/tests/test_api.py

```diff
@@ -282,3 +282,26 @@ def test_delete_message(self):
         self.assertEqual(data2["update_on"], Message.objects.get(id=2).update_on)
         self.assertEqual(data2["content"], Message.objects.get(id=2).content)
         self.assertEqual(2, Message.objects.get(id=2).token_id)
+
+
+class UserTests(APITestCase):
+    def test_user_get(self):
+        user = UserFactory()
+
+        list_url = reverse("api-user-list")
+        detail_url = reverse("api-user-detail", kwargs={"pk": user.id})
+
+        list_response = self.client.get(list_url)
+        assert list_response.status_code == 200
+
+        detail_response = self.client.get(detail_url)
```

Comment:
Perhaps compare fields here to ensure that the response and the user (from the factory) match!

Reply:
There would be no simple way to do that as the faker is random, I will fix that once I implement the patch test!